lib/tinyset.rs

```diff
@@ -254,7 +274,7 @@ pub fn tinyset_group_setref_unions<X: u64set::Fits64 + Ord + Clone>(g: &std_Grou
         {
             let rres = std_Ref::get_mut(&mut res).unwrap();
             for gr in g.iter() {
-                for v in gr.iter() {
+                for v in gr.unsorted_iter() {
```

Comment:
Change line 258 to `gr.unsorted_iter()` too?

Reply:
good point